pkgs/by-name/ri/ride/package.nix

```
@@ -109,12 +124,20 @@ buildNpmPackage rec {
 
     install -Dm644 ThirdPartyNotices.txt -t $out/share/doc/ride
 
-    mkdir -p $out/share/ride
-    cp -r locales resources{,.pak} $out/share/ride
-    makeWrapper ${lib.getExe electron} $out/bin/ride \
-      --add-flags $out/share/ride/resources/app.asar \
-      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}" \
-      --inherit-argv0
+    ${lib.optionalString (!stdenv.isDarwin) ''
```

```suggestion
  '' + lib.optionalString (!stdenv.isDarwin) ''
```
that's easier than string interpolation inside strings

---

I don't really agree with this. This is very similar to how bash's `if` statements look. I'm not using `if then else` because that would add extra levels of indentation and it doesn't really look much more readable.